tests/modules/programs/zsh/aliases.nix

```diff
@@ -0,0 +1,52 @@
+{ config, ... }:
+
+{
+  programs.zsh = {
+    enable = true;
+
+    shellAliases = {
+      test1 = "alias";
+      test2 = "alias2";
+    };
+    shellGlobalAliases = { global = "test"; };
+  };
+
+  nmt.script = ''
+    assertFileExists home-files/.zshrc
+
+    assertFileContent home-files/.zshrc ${
```

Comment:
it's too bad `assertFileContent` doesn't accept several lines, the test might be annoying to maintain. But I dont have a good suggestion. Good reflex on adding the test \o/